README.md

```diff
@@ -42,8 +42,7 @@ Ansible role: https://github.com/rossmcdonald/telegraf
 
 Telegraf requires golang version 1.10 or newer, the Makefile requires GNU make.
 
-1. [Install Go](https://golang.org/doc/install) >=1.10 (1.12 recommended)
-2. [Install dep](https://golang.github.io/dep/docs/installation.html) ==v0.5.0
+1. [Install Go](https://golang.org/doc/install) >=1.11 (1.12 recommended)
```

Comment:
We should just remove the 1.10 tests if we bump this, I assume that it's just not possible to `go mod download` in 1.10?

I think this looks fine, but one thing I always check when updating Go is the operating systems they removed support for:

> As announced in the Go 1.10 release notes, Go 1.11 now requires OpenBSD 6.2 or later, macOS 10.10 Yosemite or later, or Windows 7 or later; support for previous versions of these operating systems has been removed.

Reply:
Yeah I had to add the explicit `go mod vendor` to the `deps` task of the circle ci to get around this. Because by the time the workspace is moved into the go 1.10 image all the `go mod` commands don't exist and the go command is oblivious to the go mod build cache in `pkg`. So the vendor directly needs to be populated. Actually removing this step would mean we can drop the extra vendor step and make the packing and migrating of the workspace quicker as it should take up nearly half the space.